_posts/2024-06-26-2024-summer-research-students.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2024-06-26 23:42:21+00:00
+description: null
+featuredImage: assets/2023-08-10_sealab_reu_group_photo.jpg
+layout: profile-blog-post
+slug: /2024-summer-research-students
+title: 2024 Summer Research Students
+author: Sean
+categories:
+- news-and-updates
+students:
+- bio: TESTING SEAN
+  photo: assets/people/s.perry.jpg
+- bio: toothless
+  photo: assets/people/t.nightfury.webp
+- bio: Ben's bio
```

Comment:
This is on me, I need to add more detail to the example bios to get a better idea what we are looking for. 

Take a look at https://e4e.ucsd.edu/news-and-updates/2023-summer-research-students for some example bios